spot-ml/src/main/scala/org/apache/spot/spotldacwrapper/SpotLDACSchema.scala

```diff
@@ -8,12 +8,16 @@ object SpotLDACSchema {
   // modelDF columns
   val DocumentName = "document_name"
   val DocumentId = "document_id"
-  val WordNameWordCount = "word_count"
   val DocumentCount = "document_count"
   val DocumentNameWordNameWordCount = "document_word_count"
 
+  val WordNameWordCount = "word_count"
+  val WordName = "word_name"
```

Comment:
the distinction between document names and document IDs was not clear to me until I dived into the code... maybe put the type info into the column names or make the schema more explicity 

eg. DocumentLongID
DocumentStringName